test/test_framework.py

```diff
@@ -9,16 +9,25 @@
 
 from ops.framework import (
     Framework, Handle, EventSource, EventsBase, EventBase, Object, PreCommitEvent, CommitEvent,
-    NoSnapshotError, StoredState, StoredList, BoundStoredState, StoredStateData
+    NoSnapshotError, StoredState, StoredList, BoundStoredState, StoredStateData, SQLiteStorage
 )
 
+from sqlite3 import OperationalError
+
 
 class TestFramework(unittest.TestCase):
 
     def setUp(self):
         self.tmpdir = Path(tempfile.mkdtemp())
         self.addCleanup(shutil.rmtree, self.tmpdir)
 
+        default_timeout = SQLiteStorage.DB_LOCK_TIMEOUT
+
```

Comment:
Nitpick: empty line is a bit unexpected here. That's very tightly related to the next few lines.

Reply:
https://github.com/canonical/operator/pull/141/commits/f6004043ef0519edfe7dee7aace2ccab22300d4a